app/routes/docs/schema-browser/latest.$.ts

```diff
@@ -0,0 +1,19 @@
+import { Octokit } from '@octokit/rest'
+import type { DataFunctionArgs } from '@remix-run/node'
+import { redirect } from '@remix-run/node'
+
+const githubData = {
+  owner: 'nasa-gcn',
+  repo: 'gcn-schema',
+}
+const octokit = new Octokit()
+
+export async function loader({ params: { '*': path } }: DataFunctionArgs) {
+  let latestRelease
+  try {
+    latestRelease = (await octokit.rest.repos.getLatestRelease(githubData)).data
+  } catch (error) {
```

Comment:
What exception does it raise if there have been no releases? Can we make this `catch` block more specific?

Reply:
I suggest catching 404 errors but re-raising anything else.

Reply:
Is there a correct way to implement that with like ` if (error instanceof ???)`  

Or some other way to catch the 404?

Reply:
Something like that, yes.

Reply:
Any suggestions? Or examples? There is no class exported by octokit that allows for the instanceof method

Reply:
https://github.com/octokit/octokit.js#request-error-handling

Reply:
Perfect, thanks!